site/content/docs/concepts/pipelines.md

```diff
@@ -74,6 +74,8 @@ source:
   properties:
     branch: main
     repository: https://github.com/kohidave/demo-api-frontend
+    # Optional: specify the name of an existing CodeStar Connections connection.
+    # connection_name:
```

Comment:
can we add a sample name here too:
```suggestion
    # connection_name: a-connection
```
Taken from https://docs.aws.amazon.com/dtconsole/latest/userguide/connections-create-bitbucket.html

Reply:
If there's an example, then I should uncomment it, correct?

Reply:
Nevermind. Just got to your template comment below.